src/main/java/constants/SelectionOption.java

```diff
@@ -0,0 +1,26 @@
+package constants;
+
+import java.util.Arrays;
+
+public enum SelectionOption {
+    ADD_ATTENDANCE("1"),
+    EDIT_ATTENDANCE("2"),
+    GET_ATTENDANCE_HISTORY("3"),
+    CHECK_ABSENCE_USERS("4"),
+    QUIT("Q");
+    private final String option;
+
+    SelectionOption(String option) {
+        this.option = option;
+    }
+
+    public String getOption() {
+        return option;
+    }
+
+    public static SelectionOption getSelectOption(String input){
+        return Arrays.stream(SelectionOption.values()).filter(select -> select.option.equals(input))
+                .findAny().orElseThrow(()->new IllegalArgumentException("[ERROR] 잘못된 입력입니다."));
```

Comment:
한 줄에 하나의 메서드만 호출하도록 해주면 stream 코드의 가독성이 좋아질 것 같아요. 
추가적으로 코드 정렬도 잊지말고 꼭 부탁드릴게요. 😉

```suggestion
                .findAny()
                .orElseThrow(() ->new IllegalArgumentException("[ERROR] 잘못된 입력입니다."));
```

Reply:
로키 말대로 한 줄에 여러 메서드가 호출되어 가독성을 해치는거 같아요! 코드 정렬과 stream 코드 정렬 하도록 하겠습니다!